lib/repository.js

```diff
@@ -41,11 +41,27 @@ async function getUserInformation(uid){
     return userDoc.data();
 }
 
+async function addUserComment(comment){
+    await addDoc(collection(db,'comments/' + comment.movieId + '/movieComments'),comment);
```

Comment:
You can use string formatting [Like dis](https://www.freecodecamp.org/news/javascript-string-format-how-to-use-string-interpolation-in-js#expressions) if you want. It's a cool feature.
```suggestion
    await addDoc(collection(db, `comments/${comment.movieId}/movieComments`), comment);
```